src/resources.rs

```diff
@@ -98,3 +101,41 @@ pub struct SecretRef {
     pub name: String,
     pub key: String,
 }
+
+#[derive(CustomResource, Debug, Serialize, Deserialize, Default, Clone, JsonSchema)]
+#[kube(
+    group = "sinker.influxdata.io",
+    version = "v1alpha1",
+    kind = "SinkerContainer",
+    namespaced,
+    schema = "disabled"
+)]
+#[serde(rename_all = "camelCase")]
+pub struct SinkerContainerSpec {}
+
+const MANUAL_SCHEMA: &str = r#"
```

Comment:
had to jump through these hoops to get the `x-kubernetes-preserve-unknown-fields` field in there. taken from kubers example code